.github/workflows/bitcoin.yml

```diff
@@ -0,0 +1,76 @@
+name: bitcoin-ci
+
+on:
+  push:
+jobs:
+  build:
+    runs-on: windows-latest
+    env:
+        PYTHONUTF8: 1
+        QT_DOWNLOAD_URL: 'https://github.com/sipsorcery/qt_win_binary/releases/download/v1.4/Qt5.9.8_x64_static_vs2019.zip'
+        QT_DOWNLOAD_HASH: 'f285cbb02bec3b3f3cc2621e3fa7d5edf0d6a66fa30c57859e583acda954ea80'
+        QT_LOCAL_PATH: 'C:\Qt5.9.8_x64_static_vs2019'
+        VCPKG_INSTALL_PATH: "$env:VCPKG_INSTALLATION_ROOT/installed"
+        PLATFORM: x64
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/cache@v1
```

Comment:
The cache is only 400 MB. Is this enough? https://github.com/actions/cache#cache-limits

Reply:
It is for the vcpkg packages. It's not for the Qt binaries. In the case of the latter it's only taking around 90s to download and extract so the cache isn't essential. In the case of the former it's taking around 30 minutes to install the packages so the cache there halves the time of the job,